cyences_app_for_splunk/default/transforms.conf

```diff
@@ -10,6 +10,11 @@ max_matches = 1
 filename = ip_blocked_list.csv
 max_matches = 1
 
+# Asset Inventory
+[cs_all_assets]
+external_type = kvstore
+collection = cs_all_assets_collection
+fields_list = _key, time, host, uuid, asset_id, asset_hostname, asset_ip, asset_mac_address, os, state, last_scan_time, vendor_product
```

Comment:
can we use the same order as savedsearches.conf. I think that will make development easier in the future changes.